From bf36d3f3ddb55505369347e5133785e32cb5d2fd Mon Sep 17 00:00:00 2001
From: Takuro Ashie <ashie@clear-code.com>
Date: Wed, 18 Dec 2024 16:02:09 +0900
Subject: [PATCH] Load libv4l2.so.0 instead of libv4l2.so

SO version should be added to keep binary compatibility, and to work
without development pacakge.

Signed-off-by: Takuro Ashie <ashie@clear-code.com>
---
 media/gpu/v4l2/v4l2_device.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/media/gpu/v4l2/v4l2_device.h b/media/gpu/v4l2/v4l2_device.h
index 8ee721ce592b8..92f89dbc683bf 100644
--- a/media/gpu/v4l2/v4l2_device.h
+++ b/media/gpu/v4l2/v4l2_device.h
@@ -685,9 +685,9 @@ class MEDIA_GPU_EXPORT V4L2Device
 
   inline static constexpr char kLibV4l2Path[] =
 #if defined(__aarch64__)
-      "/usr/lib64/libv4l2.so";
+      "/usr/lib64/libv4l2.so.0";
 #else
-      "/usr/lib/libv4l2.so";
+      "/usr/lib/libv4l2.so.0";
 #endif
 
   // Returns true iff libv4l2 should be used to interact with the V4L2 driver.
-- 
2.34.1

